docker/Dockerfile.ci-operator-buildroot

```diff
@@ -21,14 +22,24 @@ RUN mkdir /go && \
     chgrp -R 0 /go && \
     chmod -R g=u /go
 
+# Configure npm package manager (installed below) via environment variables.
+# https://docs.npmjs.com/cli/v8/using-npm/config#environment-variables
+ENV npm_config_cache=/go/.npm \
+    npm_config_update_notifier=false
+
 # Install additional build dependencies.
 # Node.js (and npm) is installed via the OS provided nodejs module stream.
-# npm is updated to latest version prior to installing any global npm packages.
-RUN dnf module enable nodejs:16 -y && \
-    dnf module install nodejs:16 -y && \
-    npm config -g set cache /go/.npm && \
-    npm install -g --no-update-notifier npm && \
-    npm install -g --no-update-notifier yarn
+RUN dnf module enable -y nodejs:16 && \
+    dnf module install -y nodejs:16 && \
+    npm install -g yarn
```

Comment:
Removed the command for updating npm itself, since it wasn't working as expected.

Installing `nodejs` package also pulls in the `npm` package, i.e. these are installed as system packages from Fedora RPM repository. Therefore, trying to update npm via `npm install -g npm` has no real effect.

In other words, we simply use the available npm version provided as part of the `nodejs` module stream.